lib/ae_mdw/dry_run/runner.ex

```diff
@@ -0,0 +1,26 @@
+defmodule AeMdw.DryRun.Runner do
+  @moduledoc """
+  Simulates transactions with dry run.
+  """
```

Comment:
Not sure I fully understand what this mod does, perhaps you can explain it better? And why do we need to hard-code a pk also?

Reply:
It's the concept of dry-run but after your question added info to be more clear. Better to understand?